package.json

```diff
@@ -6,6 +6,9 @@
   "repository": "muut/riotjs",
   "author": "Muut, Inc. and other contributors",
   "license": "MIT",
+  "scripts": {
+    "test": "make test"
```

Comment:
True but I thought this PR was to get tests running in the new framework. Travis is a next step so it would seem this is premature as well.